.github/workflows/build-and-test.yml

```diff
@@ -97,6 +97,22 @@ jobs:
         with:
           files: ./target/scala-3.3.6/coverage-report/cobertura.xml
 
+  test-ingest-integration:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout current branch
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Setup Java
+        uses: actions/setup-java@v4
+        with:
+          distribution: temurin
+          java-version: 21
+          cache: sbt
+      - name: Run integration tests
+        run: export DOCKER_BUILDKIT=1; ./sbtx Docker/publishLocal; ./sbtx ingestIntegration/test
```

Comment:
For the other e2e/it tests we have a job which parses the xunit xml report, can you please add this too?